systemd-inhibit: add page (#10879)
tldr-pages/tldr: pages/linux/systemd-inhibit.md

# systemd-inhibit

> > Prohibit system from entering certain power states.

Reviewer: ```suggestion
> Prohibit the system from entering certain power states.
```

> Inhibitor locks may be used to block or delay system sleep and shutdown requests as well as automatic idle handling.
> More information: <https://www.freedesktop.org/software/systemd/man/systemd-inhibit.html>.

> - List all active inhibition locks instead of acquiring one:

Reviewer 2: ```suggestion
- List all active inhibition locks:
```

`systemd-inhibit --list`

> - Block system shutdown with sleep command:

Reviewer 3: ```suggestion
- Block system shutdown for {{5}} seconds with sleep command:
```

> - Block system shutdown for {{5}} seconds with sleep command:

Reviewer: ```suggestion
- Block system shutdown for a specified number of seconds with the `sleep` command:
```

> `systemd-inhibit --who $USER --what shutdown sleep {{5}}`

Reviewer 2: Moved to another example to describe the option.
```suggestion
`systemd-inhibit --what shutdown sleep {{5}}`
```

> - Keep system from sleeping or idling until after download is finished:

Reviewer: ```suggestion
- Keep the system from sleeping or idling until after the download is completed:
```

> - Keep the system from sleeping or idling until after the download is completed:

Reviewer 2: ```suggestion
- Keep the system from sleeping or idling until the download is complete:
```

`systemd-inhibit --what sleep:idle wget {{https://example.com/file}}`

> - Ignore lid close switch until script exits:

Reviewer 2: ```suggestion
- Ignore lid close switch until the script exits:
```

`systemd-inhibit --what sleep:handle-lid-switch {{path/to/script}}`

- Ignore power button press while command is running:

> `systemd-inhibit --what handle-power-key {{command}}`

Reviewer 2: ```suggestion
`systemd-inhibit --what handle-power-key {{command}}`

- Describe who and why created the inhibitor (default: the command and its arguments for `--who` and `Unknown reason` for `--why`):

`systemd-inhibit --who {{$USER}} --why {{reason}} --what {{operation}} {{command}}`
```
